File: src/app/model/private-talk-container.model.ts
import { MessageCountModel } from './message-count.model'
import { PrivateTalkReceiver } from './private-talk-receiver.model'
import { PrivateTalkTeamReceiver } from './private-talk-team-receiver.model'
import { PrivateTalk } from './private-talk.model'

export class PrivateTalkContainerModel {
    pTalks: PrivateTalk[] = []
    messageCounts: MessageCountModel[]= []
    ptrs: PrivateTalkReceiver[]= []
    pttrs: PrivateTalkTeamReceiver[]= []
    totalUnReadCount:number

}

File: Models/ViewModels/PrivateTalkContainerModel.cs
using System;

namespace XYZToDo.Models.ViewModels
{
    public class PrivateTalkContainerModel
    {
        public PrivateTalk[] pTalks { get; set; }
        public MessageCountModel[] messageCounts { get; set; }
        public PrivateTalkReceiver[] ptrs { get; set; }
        public PrivateTalkTeamReceiver[] pttrs { get; set; }
        public int totalUnReadCount { get; set; }
    }
}
